src/numerics.jl

```diff
@@ -12,6 +12,11 @@ function evalf(b::Basic, bits::Integer=53, real::Bool=false)
     if status == 0
         return c
     else
+        # replace any True/False with 1/0
+        m = CMapBasicBasic()
+        m[True] = ONE; m[False] = ZERO
```

Comment:
Can we make this map a global variable instead of creating it every time?

Reply:
I need your help with that. I was trying, but kept getting segfaults. The construction of `Basic` objects is a bit different than that of `CMapBasicBasic`, so I was having issue initializing the map in `__init__`.

One thing about what I have here that bothers me is I'd like to cheaply test if an expression has a `BooleanAtom` and then try to evaluate that expression after substitution with `evalf`, but it seemed easier to do the substitution and then compare the result to the original. I wasn't sure if substituting in a map with no matches would always return the original expression, as the test has that assumption.

Reply:
I just checked in what I thought I'd tried earlier and it seems to not cause an issue. I'll add a direct test to be sure.